app.py

```diff
@@ -32,16 +33,17 @@
 
 def tryExecute(f, *a, **k):
     """
-    Execute the function, catching exceptions and logging as an error. Return 
-    False to indicate an exception. 
+    Execute the function, catching exceptions and logging as an error. Return
+    False to indicate an exception.
 
     Args:
         f (func): The function.
-        *a (mixed): Optional positional arguments
-        **k (mixed): Optional keyword arguments.
+        *a (value tuple): Optional positional arguments.
+        **k (kw tuple): Optional keyword arguments.
```

Comment:
I've put this as a type in a few places.  Is this correct and a good way to write this?

Reply:
`dict` is better. Probably just 

```
*a (tuple): ...
**k (dict): ...
```